ctapipe_io_lst/event_time.py

```diff
@@ -180,7 +180,7 @@ def __call__(self, tel_id, event):
 
         # first event and values not passed
         else:
-            if not ucts_available:
+            if not ucts_available and self.timestamp == 'ucts':
```

Comment:
No! The code below uses ucts to create tib and dragon time. So you cannot only raise the exception when `self.timestamp == 'ucts'`.

This codepath needs to run for tib and dragon timestamps and simply does not work if ucts is not available.

Reply:
Ok @maxnoe , I did not dig inside all these methods, please suggest you a way to process files without ucts data, for the moment it is not possible as far as I understand (perhaps I am missing a way?)

Reply:
We discussed it in slack: add an option to the source to not fill the timestamp.

Reply:
That is doesn't work, because I need the timestamp and for those runs I have it from the tib

Reply:
why I should not use it?

Reply:
There is now way to get a correct timestamp without ucts.

Reply:
So the question is for what do you need timestamps?

Reply:
> There is no way to get a correct timestamp without ucts. See discussion here:
> [cta-observatory/cta-lstchain#581](https://github.com/cta-observatory/cta-lstchain/issues/581)

They were the only one accessible for several months, so we decided to through everything away? 
Then, the problem is solved. I do not think it makes sense to analyse data for which we do not fill the timestamp. 

Only, we must decide a new time reference run, because the 1625 can not be used anymore. What do you think @moralejo ?

Reply:
> I do not think it makes sense to analyse data for which we do not fill the timestamp.

I agree. I usually do not do it and when I do it just for completeness I note it down in the analysis logbook we have.

Only, we must decide a new time reference run, because the 1625 can not be used anymore. What do you think @moralejo ?

I've been producing the time calibration files every day for some time now. When it is now possible I use 1625. Wanted to ask you about this back then, but never did. I hope it is not a problem.

Reply:
In any case, it seems to me that a timestamp with global offset  is better than zero timestamp

Reply:
I disagree. A wrong timestamp that looks like it could be right is worse than knowing there is no correct timestamp.

Reply:
We could however, allow filling the invalid timestamp `0 MJD TAI`

Reply:
Hi @morcuended,

> I've been producing the time calibration files every day for some time now. When it is not possible I use 1625. Wanted to ask you about this back then, but never did. I hope it is not a problem.

These corrections are supposed to be stable with time and change only of there is some hardware change. So  the idea was to keep one reference file  for each hardware period (and we started with 1625) in order to be sure that the time coefficients were properly estimated and the camera setup was ok. But it is true that at end we remained fixed to it. Did I understand correctly that you are now using the time coefficients of the calibration run of each night?

Reply:
> Did I understand correctly that you are now using the time coefficients of the calibration run of each night?

Correct

Reply:
> I disagree. A wrong timestamp that looks like it could be right is worse than knowing there is no correct timestamp.

It is only an offset of some seconds (for these commissioning data it is not so critical) and if we ask expressively to use tib, we take a risk under control (it is not going to tib by default).

Reply:
I agree with @FrancaCassol, and we do not want to be unable to process some data with the new release.
If I understand properly, the example Franca mentioned is that of a calibration run. We will not need accurate timing for such a run (or even any timing at all, since for now we will just use the whole subrun's interleaved events to locate stars and modify cleaning levels). But definitely we want to be able to process it.

Reply:
This is what I proposed in the first place, an option not to fill timing info (or fill clearly invalid), but having timestamps that is wrong by 8 to 120 seconds is something I would not do.

Reply:
But then, @FrancaCassol, why is the timestamp needed?

Reply:
If it is about relative time, you can just directly access the counter values from  `event.lst.evt`.

But filling `trigger.time` with a wrong absolute timestamp is not a good idea.

Reply:
They are used in the code: there is an option to select the calibration statistics  based on a time interval not on the number of the events. In this case a global shift does not hurt, because only relative times are considered.

Reply:
Let's say, the important thing is to fill a timestamp. I would keep the tib value (knowing that I am doing it and the risks I take) but to de-block the situation a zero value will also work